src/Libraries/Nop.Data/Migrations/UpgradeTo460/DataMigration.cs

```diff
@@ -5,19 +5,22 @@
 using System.Linq;
 using FluentMigrator;
 using Nop.Core;
+using Nop.Core.Domain.Catalog;
 using Nop.Core.Domain.Common;
 using Nop.Core.Domain.Configuration;
 using Nop.Core.Domain.Customers;
 using Nop.Core.Domain.Directory;
 using Nop.Core.Domain.Localization;
 using Nop.Core.Domain.Logging;
+using Nop.Core.Domain.Media;
 using Nop.Core.Domain.Messages;
 using Nop.Core.Domain.ScheduleTasks;
 using Nop.Core.Domain.Security;
+using Nop.Data.Migrations.UpgradeTo460.Domain;
 
 namespace Nop.Data.Migrations.UpgradeTo460
 {
-    [NopMigration("2022-02-03 00:00:00", "4.60.0", UpdateMigrationType.Data, MigrationProcessType.Update)]
+    [NopMigration("2022-02-03 00:00:00", "DataMigration for 4.60.0", MigrationProcessType.Update)]
```

Comment:
I don’t understand why these changes were made, especially since they break the logic of running this migration every time the debug mode is set in the studio (see https://github.com/nopSolutions/nopCommerce/blob/061ea8c69312bab8c7b08951d54858f23af964d5/src/Libraries/Nop.Data /Migrations/MigrationManager.cs#L125-L129)